webapp/lib/controller.dart

```diff
@@ -128,25 +143,39 @@ void command(UIAction action, Data actionData) {
       ProjectConfigurationData projectConfigurationData = actionData;
       saveProjectConfiguration(projectConfigurationData.config);
       break;
+    case UIAction.savePackageConfiguration:
+      savePackageConfiguration();
+      break;
   }
 }
 
+enum NavAction {
+  allProjects,
+  dashboard
+}
+
 void loadAuthView() {
+  view.navView.navActions[NavAction.allProjects](false);
+  view.navView.navActions[NavAction.dashboard](false);
```

Comment:
Sorry - this isn't really what I meant in my previous comment.

I'd like there to be a single method call here, something like:
```
view.navView.showParent(NavAction.none);
```
So instead of the logic for setting the visibility for each nav action being in the controller, it should be in the view. Same for the lines below.

(and it occurs to me that we should have a `NavAction.none` when we don't want to show anything)

Reply:
Understood. Re-implemented as specified